src/options/smt_options.toml

```diff
@@ -9,6 +9,58 @@ name   = "SMT Layer"
   default    = "false"
   help       = "eliminate functions by ackermannization"
 
+[[option]]
+  name       = "computePartitions"
+  category   = "expert"
+  long       = "compute-partitions=N"
+  type       = "uint64_t"
+  default    = "0"
+  help       = "make n partitions"
+
+[[option]]
+  name       = "numChecks"
+  category   = "expert"
+  long       = "num-checks=N"
+  type       = "uint64_t"
+  default    = "1"
+  help       = "number of standard effort checks until partitioning"
+
+[[option]]
+  name       = "partitionConflictSize"
+  category   = "expert"
+  long       = "conflict-size=N"
+  type       = "uint64_t"
+  default    = "0"
+  help       = "number of literals in a cube"
+
+[[option]]
+  name       = "partitionStrategy"
+  category   = "expert"
+  long       = "partition-strategy=X"
+  type       = "std::string"
```

Comment:
Instead of using an `std::string`, it would be better to use a "mode" option. See [here](https://github.com/cvc5/cvc5/blob/master/src/options/decision_options.toml#L4-L27) for an example.